src/main/java/org/broadinstitute/hellbender/utils/baq/BAQ.java

```diff
@@ -627,7 +602,7 @@ public BAQCalculationResult calcBAQFromHMM(GATKRead read, byte[] ref, int refOff
                     readI += l;
                     break;
                 case D : refI += l; break;
-                case M :
+                case M : case EQ: case X:
```

Comment:
this was fixed in https://github.com/broadinstitute/gsa-unstable/pull/807
Not much to say here - all 3 operators are equivalent here